source/types/types.go

```diff
@@ -6,4 +6,5 @@ const (
 	LocalScheme       = "local"
 	HTTPScheme        = "http"
 	HTTPSScheme       = "https"
+	OCIScheme         = "oci"
```

Comment:
We already have `--output type=oci`, so `oci` might be fine.

Or we might want to consider renaming `--output type=oci` to `--output type=oci-layout` too.

Reply:
Or `oci-archive` maybe

Reply:
Atm. this string isn't even visible to the user so some verbosity should be fine in here.

Reply:
`--output type=oci` gives an OCI tarball. It is the same format, except in a tarball. Which leads to the question of whether we should support tarball input as well as directory (and maybe directory layout in addition to tarball).

It is much easier to do tarball, because it is a stream, no need to worry about access to local directories.

I don't think they all need to be solved now, but we should get the nomenclature correct so we have options in the future without breaking things.

We can keep just a single scheme/type `oci` (or `oci-layout`, if we prefer) for all of it, with the below:

* source directory: `oci://path/to/layout/dir`
* source tar file: `oci://path/to/tarball.tar` (future, would depend on if it is a file of type tar or not)
* source tar stream: `oci://` or `oci://-` or even `oci://stdin`
* output directory: `--output type=oci,dest=path/to/layout/dir`
* output tar file: `--output type=oci,dest=path/to/output.tar`
* output stdout: `--output type=oci`

The only really weak point there is output directory, as it might not exist, so how do we know if they want a tar file or a layout directory? We could say, "if dest ends in `.tar`, we will write a tar file, else a directory, and if you really want to write to a tar file that does not end in `.tar`, then write to stdout and save the output as you wish.